domains/sushi.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "sushi-ae",
+        "email": "frapdotpng@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "sushi-ae.github.io"
```

Comment:
Just redirects to our site. Please have an actual site.

Reply:
why does it do that, I didn't even set an alternate domain yet

Reply:
even if I unpublish the site, sushi-ae.github.io still redirects to is-a.dev

Reply:
any ideas why this happening? or how to potentially fix it?